projects.csv

```diff
@@ -0,0 +1,2 @@
+projects
```

Comment:
this file is for?

Reply:
give template (I have rename with one .exemple)